#159965310 mockups for article likes and dislikes

meshack-mbuvi opened the pull request:
#### what does this pull request do
- Adds mockups for article likes and dislikes

#### Description of tasks to be completed
- Have buttons alongside an article for a user to click to like or dislike the article 
- Show the number of likes and dislikes for a given article
- have a navigation bar for different sections

#### How should this be tested manually
- Clone this repository
- Run the command `git checkout ft-likes-dislikes-mockups-159965310`
- change directory to `ui-mockups` by executing `cd ui-mockups` in your terminal
- Do the following commands to set your environment for testing using your terminal
    - Install [node](Download | Node.js)
    - Run `npm install` to install all packages defined in `package.json` file
    - npm install gulp-cli -g
    - npm install gulp -D
- After executing above commands,type `gulp` and hit `[Enter]` button
- On your browser, visit the follwing pages
    - http://localhost:3000/articles.html
    - http://localhost:3000/article.html
    - http://localhost:3000/myarticles.html
#### Any background context you want to provide?
Having mockups is important in every API development because it helps to visualize how the user interface will look like in the finished product.

#### What are the relevant pivotal tracker stories?
[Finishes #159965310]

#### screenshots
![image](https://user-images.githubusercontent.com/9263906/45613674-bc6a0100-ba6f-11e8-940a-2992064146c9.png)

meshack-mbuvi committed:
[feat]: add mockups for article likes and dislikes

- Have buttons for user to click to like or dislike an article
- have a navigation bar for different sections

[Finishes #159965310]

mikenthiwa commented:
@meshack-mbuvi The button is not visible.
I noticed that the space betwwen like and dislike button is small increase the margin between them in article.html.
Also remove the background image.

pabishai commented on ui-mockups/src/articles.html:
```diff
@@ -87,20 +119,24 @@ <h6><a href="article.html">Donec id elit non mi porta</a></h6>
+                                By <a class="ah-author-link" href="#">Mbuvi</a>
                             </small>
-                            <p class="d-flex justify-content-between">
+                            <p class="d-flex justify-content-between article-footer">
```
@meshack-mbuvi Is the lower border necessary. Also I think the colors do not really blend into the existing template we have.

meshack-mbuvi replied:
Which color do you recommend for the likes and dislikes @PAbishai ?

pabishai replied:
Maybe space the icons and the number of likes a bit and tint the color a bit to make it lighter and blend in more with the other colors

pabishai reviewed (changes requested):
Reviewed your PR

micahoriaso commented:
@meshack-mbuvi please remove all underlines from under the articles and the underline under the likes and dislikes buttons. Also, remove the background hover on the likes and dislikes button instead change the colour of the icons themselves. All the undelines make the articles page look crowded, therefore giving the user too much information making the user distracted from the main point which is reading the article.

meshack-mbuvi commented:
@micahoriaso Give reasons why I should do so

Bileonaire commented:
@meshack-mbuvi good work.
I suggest that you remove the wordings in the like and dislike buttons because the icons serve the purpose.

meshack-mbuvi commented:
@Bileonaire I understand your concern, I got the implementation from facebook. Was thinking that some users may not get the full meaning for a thumb up without the words.Am considering your feedback.

pngg commented:
Looks good!

meshack-mbuvi committed:
[design]: improve user interface for the articles

- fade the color of border on each article
- change color of likes and dislikes icons
- remove words from the likes/dislikes icons

[Finishes #159965310]

micahoriaso commented:
Looks good to me.

micahoriaso commented:
> @meshack-mbuvi please remove all underlines from under the articles and the underline under the likes and dislikes buttons. Also, remove the background hover on the likes and dislikes button instead change the colour of the icons themselves. All the undelines make the articles page look crowded, therefore giving the user too much information making the user distracted from the main point which is reading the article.

Looks good to me

meshack-mbuvi committed:
[design]: fix likes/dislikes after merge
